jnigen/bin/jnigen.dart

```diff
@@ -3,8 +3,15 @@
 // BSD-style license that can be found in the LICENSE file.
 
 import 'package:jnigen/jnigen.dart';
+import 'package:jnigen/src/logging/logging.dart';
 
 void main(List<String> args) async {
-  final config = Config.parseArgs(args);
+  Config config;
+  try {
+    config = Config.parseArgs(args);
+  } on ConfigException catch (e) {
+    log.fatal(e);
```

Comment:
You might want to also log the stack-trace.

Reply:
`ConfigException` exists to point out to the user that configuration is wrong.

We could `log.fatal` in parsing code, but throwing an exception allows us to test.

Reply:
I think that's fine.

You could make it:

```dart
} catch(e, stacktrace) {
  log.fatal('$e\n$stacktrace');
}
```

Reply:
Ah, fair enough. I didn't actually take a look at the error messages produced by this.